src/js/utils/captions.js

```diff
@@ -250,10 +250,11 @@ export const areSubsOn = (el) => {
  *
  * This was originally in media-captions-button.
  *
- * @param {HTMLElement} el - An HTMLElement that has caption related attributes on it.
+ * @param {HTMLElement & { mediaSubtitlesList?: any[] }} el - An HTMLElement that has caption related attributes on it.
  * @param {boolean} [force] - An optional boolean that will force captions to the given state. True for on and false for off
+ * @param {boolean} [noSubtitlesPref = false] - An optional boolean that will force captions to the given state. True for on and false for off
  */
-export const toggleSubsCaps = (el, force) => {
+export const toggleSubsCaps = (el, force, noSubtitlesPref = false) => {
```

Comment:
This might be a different topic of old wounds but remind me why this can't just be handled by the controller, and has to be in the UI components?

Reply:
Per out of band convo, this was because of prior hesitancy to introduce a "toggle" subs/cc request event. We'll be introducing it now to avoid these acrobatics.

Reply:
No longer an issue with the refactor that adds the toggle request event.